Makefile

```diff
@@ -176,6 +176,8 @@ clean: clean_package
 	$(MAKE) -C $(CC_WEB) clean
 
 clean_package: clean_plist_to_html clean_tu_collector clean_report_converter clean_report_hash clean_statistics_collector
+	$(MAKE) -C $(CC_ANALYZER) clean_package
+	$(MAKE) -C $(CC_WEB) clean_package
```

Comment:
Do we need these cleanups here? They remove the npm packages too and they needs to be downloaded again.

Reply:
Yes, we need it. This one will remove the dist directory only and it will not remove the node_modules directory:
https://github.com/Ericsson/codechecker/blob/eb039435f9a335ed2ec8e69204a070a02fe22e69/web/Makefile#L113-L116
Only the clean target will remove the node_modules directory:
https://github.com/Ericsson/codechecker/blob/eb039435f9a335ed2ec8e69204a070a02fe22e69/web/Makefile#L121-L123